sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Release History
 ## 1.1.0-beta.3 (Unreleased)
+### New Features
+- Added support FCM push notifications
+    - ChatAsyncClient:
+        - Added `startPushNotifications`
+        - Added `stopPushNotifications`
+        - Added `handlePushNotification`
+        - Added `addPushNotificationHandler`
+        - Added `removePushNotificationHandler`
+    - ChatClient:
+        - Added `startPushNotifications`
+        - Added `stopPushNotifications`
+        - Added `handlePushNotification`
+        - Added `addPushNotificationHandler`
+        - Added `removePushNotificationHandler`
+     - Added `ChatPushNotification` as the type for passing push notification payload
+     - Added `PushNotificationCallback` as the type for customized push notification handler
+     - Added instructions in README.md in sample app for steps to setup FCM push notifications
+     - Added code in sample app for FCM push notification end to end scenario
+- Added `startRealtimeNotifications(Context context)` to ChatAsyncClient
+- Added `startRealtimeNotifications(Context context)` to ChatClient
```

Comment:
We are using a different header now for new features. Also, you might want to make it clear these are methods users can call.
```suggestion
### Features Added
- Added support FCM push notifications:
    - Added methods to `ChatAsyncClient`:
        - `startPushNotifications`
        - `stopPushNotifications`
        - `handlePushNotification`
        - `addPushNotificationHandler`
        - `removePushNotificationHandler`
    - Added methods to `ChatClient`:
        - `startPushNotifications`
        - `stopPushNotifications`
        - `handlePushNotification`
        - `addPushNotificationHandler`
        - `removePushNotificationHandler`
     - Added `ChatPushNotification` as the type for passing a push notification payload.
     - Added `PushNotificationCallback` as the type for a customized push notification handler.
     - Added instructions in README.md in sample app for steps to setup FCM push notifications.
     - Added code in sample app for FCM push notification end to end scenario.
- Added `startRealtimeNotifications(Context context)` to `ChatAsyncClient`.
- Added `startRealtimeNotifications(Context context)` to `ChatClient`.
```